LICENSE

```diff
@@ -0,0 +1,40 @@
+SPRYKER SYSTEMS GMBH EVALUATION LICENSE AGREEMENT
+
```

Comment:
* All `Eco` modules a published under the MIT license.
* Use as a source https://github.com/spryker-eco/oauth-azure/blob/master/LICENSE.

Reply:
Resolved.